html/contributors.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/index.css"> 
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.5.1/css/all.css">
+    <title>Contributors</title>
+</head>
+
+<body class="contributors-body">
+    <header>
```

Comment:
We'll want to include a path to the Header logo here as well, so the nav doesn't jump over to the left. Would be great if you could create a new issue for this. Let me know if you want to be assigned - should be a quick fix.

Reply:
Yes, please assign me! Or the person who did all the work on the header. Glad I changed my index.js haha.